contracts/SponsorFaucet.sol

```diff
@@ -0,0 +1,92 @@
+pragma solidity 0.5.11;
+pragma experimental ABIEncoderV2;
+
+import "@openzeppelin/contracts/ownership/Ownable.sol";
+import "@openzeppelin/contracts/lifecycle/Pausable.sol";
+import "./Lib/ReentrancyGuard.sol";
+
+contract SponsorWhitelistControl {
```

Comment:
Duplicate with InternalContract.sol? Try simply import it.

Reply:
fix by import